L/libcxxwrap_julia/build_tarballs.jl

```diff
@@ -12,10 +12,6 @@ unpack_target = is_yggdrasil ? "" : "libcxxwrap-julia"
 # Collection of sources required to complete build
 sources = [
     GitSource(git_repo, "7959f09947f95e67ee094a2ee3fb8b7f9be696f8", unpack_target=unpack_target),
-    ArchiveSource("https://julialang-s3.julialang.org/bin/linux/armv7l/1.3/julia-1.3.1-linux-armv7l.tar.gz", "965c8fab2214f8ce1b3d449d088561a6de61be42543b48c3bbadaed5b02bf824"; unpack_target="julia-arm-linux-gnueabihf"),
-    ArchiveSource("https://julialang-s3.julialang.org/bin/linux/x64/1.3/julia-1.3.1-linux-x86_64.tar.gz", "faa707c8343780a6fe5eaf13490355e8190acf8e2c189b9e7ecbddb0fa2643ad"; unpack_target="julia-x86_64-linux-gnu"),
-    ArchiveSource("https://github.com/Gnimuc/JuliaBuilder/releases/download/v1.3.0/julia-1.3.0-x86_64-apple-darwin14.tar.gz", "f2e5359f03314656c06e2a0a28a497f62e78f027dbe7f5155a5710b4914439b1"; unpack_target="julia-x86_64-apple-darwin14"),
-    ArchiveSource("https://github.com/Gnimuc/JuliaBuilder/releases/download/v1.3.0/julia-1.3.0-x86_64-w64-mingw32.tar.gz", "c7b2db68156150d0e882e98e39269301d7bf56660f4fc2e38ed2734a7a8d1551"; unpack_target="julia-x86_64-w64-mingw32"),
```

Comment:
There is some code below which hardcodes the version as 1.3.1, and which special cases ming32/darwin -- that needs to be adjusted.

Reply:
One could also enable the FreeBSD platform, I guess? Maybe hardcoding a list of platforms isn't necessary anymore at all?